Dockerfile

```diff
@@ -2,12 +2,18 @@
 # Use of this source code is governed by an Apache2
 # license that can be found in the LICENSE file.
 
-FROM scratch
+FROM alpine:latest
 
 MAINTAINER Torin Sandall <torinsandall@gmail.com>
 
-ADD /opa_linux_GOARCH /opa
+ENV BIN /usr/bin/opa
 
-ENTRYPOINT ["/opa"]
+RUN apk add --update ca-certificates &&\
+    rm -rf /var/cache/apk/* &&\
+    update-ca-certificates
 
-CMD ["run"]
```

Comment:
Is there a particular reason for this? The default command was specified so that `docker run -i --rm openpolicyagent/opa` just did the right thing.